setup.py

```diff
@@ -59,7 +51,7 @@ def _get_data_files():
 
 setuptools.setup(
     name=NAME,
-    version=_get_version(),
+    version="28.0.dev1",
```

Comment:
Should we consider using a dummy version here ? My concern is that if in the future we bump the version past 28, this might get misleading again as we have today with the `20.3`  version we see on the logs

Reply:
+1 to more obviously fake version. Maybe something over 9000? `9001`? :upside_down_face:

Reply:
The American translation is wrong - in Portuguese, as well as the original in Japanese, it is over 8000
I will set it to 8001

Reply:
TIL!